src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -502,14 +502,14 @@ public IActionResult GetHistory([FromQuery] WalletHistoryRequest request)
                             }
 
                             // No need for further processing if the transaction itself is the output of a staking transaction.
-                            if (transaction.IsCoinStake != null)
+                            if (transaction.IsCoinStake == true)
```

Comment:
```suggestion
                            if (transaction.IsCoinStake)
```

Reply:
That won't work for a `null`-able boolean.